lib/fog/aws/requests/compute/run_instances.rb

```diff
@@ -221,6 +232,7 @@ def run_instances(image_id, min_count, max_count, options = {})
               'associatePublicIP'     => options['associatePublicIP'] || false,
               'architecture'          => 'i386',
               'blockDeviceMapping'    => block_device_mapping,
+              'hibernationOptions'    => hibernation_options,
```

Comment:
Hey, I'm not sure why the integration isn't working right, but travis-ci is reporting an error here: https://travis-ci.org/github/fog/fog-aws/jobs/699781152#L413. In particular, `hibernation_options` is referred to is this mock version of the method without having previously been defined. We may need to either refer directly to the options hash or do something similar to the delete above. Does that make sense? Thanks!

Reply:
Hopefully this change should do it.  Tested successfully running local mock test.